### ISSUE TITLE
refresh after registry import

### ISSUE THREAD

**Issue by michel-cote** [Open-Shell/Open-Shell-Menu#393]
**Is your feature request related to a problem? Please describe.**
Based on different actions by a user, we are importing Open-Shell Startmenu registry settings.   How do we refresh the StartMenu immediately to reflect the new registry settings, without forcing the user to logoff and then back on again?  

**Describe the solution you'd like**
Do we need a new commandline option of "/refresh"?  or is there a windows commandline to refresh the cached version of the registry?

**Describe alternatives you've considered**
We have tried killing the openshell startmenu exe process and then re-running it, but it seems this does not work.  It seems that a cached version of the registry settings is being used??

**Comment by ge0rdi** [comment]
Can't you just import settings from XML config file?
Instead of directly changing registry.

**Comment by michel-cote** [comment]
Is there a commandline function to import XML config files?

**Comment by ge0rdi** [comment]
Try to use:
`"C:\Program Files\Open-Shell\StartMenu.exe" -xml <path-to-xml-config>`

**Comment by michel-cote** [comment]
I used the settings gui to backup to an XML and then imported it to test that the XML is valid, and it works through the settings GUI backup option.  But when I try to import using the commandline as you specified, I get this error:
"XML parsing error: The system cannot locate the object specified."

**Comment by ge0rdi** [comment]
How exactly do you run the command?
Do you provide full path to XML file?

**Comment by michel-cote** [comment]
No, we just set the current directory to the location of the XML file.  We tried again and specified the full path and it works perfect.  Thanks.   I'll finish testing our script and then post it up here if anyone is interested - the script creates a custom settings file (now XML!) per user based on their Windows security groups that they belong to.   On an RDP server, we can now add an active user to a new Windows group and their StartMenu auto-updates to add links to folders and/or programs they now have access to, WITHOUT having to force the user to logoff/logon.  This makes it easy to manage custom startmenus for all users on an RDP server based on their Windows security groups - works for domain and workgroup users.
Thanks again.
Should this -xml commandline option be added to the Help file and/or wiki?
